lbrynet/blob/blob_file.py

```diff
@@ -142,33 +141,6 @@ def is_downloading(self):
             return True
         return False
 
-    def read(self, write_func):
-        """
-        This function is only used in StreamBlobDecryptor
-        and should be deprecated in favor of open_for_reading()
-        """
-        def close_self(*args):
-            self.close_read_handle(file_handle)
-            return args[0]
-
-        file_sender = FileSender()
-        reader = HashBlobReader_v0(write_func)
-        file_handle = self.open_for_reading()
-        if file_handle is not None:
-            d = file_sender.beginFileTransfer(file_handle, reader)
-            d.addCallback(close_self)
-        else:
```

Comment:
you can see here that it uses FileSender() so that StreamBlobDecryptor can decrypt blobs asynchronously